JsonSchema/EvaluationResults.cs

```diff
@@ -421,13 +435,13 @@ public override void Write(Utf8JsonWriter writer, EvaluationResults value, JsonS
 		if (value.Format == OutputFormat.Hierarchical || value.Parent != null)
 		{
 			writer.WritePropertyName("keywordLocation");
-			options.Write(writer, value.EvaluationPath, JsonSchemaSerializerContext.Default.JsonPointer);
+			JsonSerializer.Serialize(writer, value.EvaluationPath, options);
```

Comment:
Why did these switch back to JsonSerializer?

Reply:
Not sure.  I'll put them back.